src/classes/UTIL_CascadeDeleteLookups_TDTM.cls

```
@@ -0,0 +1,131 @@
+        Contact.SObjectType                      =>  new List<String>{'npe03__Recurring_Donation__c.npe03__Contact__c'},
+        Account.SObjectType                      =>  new List<String>{'npe03__Recurring_Donation__c.npe03__Organization__c'},
+        Opportunity.SObjectType                  =>  new List<String>{'Allocation__c.Opportunity__c'},
+        Campaign.SObjectType                     =>  new List<String>{'Allocation__c.Campaign__c'},
+        npe03__Recurring_Donation__c.SObjectType =>  new List<String>{'Allocation__c.Recurring_Donation__c'}
+    };
+
+    /*******************************************************************************************************
+    * @description Run method checks if there is a cascade deletion for the current trigger and runs the
+    * cascade method for each entry.
+    * @param listNew The list of sobjects from trigger new.
+    * @param listOld The list of sobjects from trigger old.
+    * @param triggerAction Which action triggered the current context. (Before or After; Insert, Update, 
+    * Delete, or Undelete.)
+    * @param objResult The field describe object for the current trigger set.
+    * @return dmlWrapper Objects that need to be inserted or updated after the processing of the trigger.
+    ********************************************************************************************************/
+    public override DmlWrapper run(List<SObject> newlist, List<SObject> oldlist, 
+        TDTM_Runnable.Action triggerAction, Schema.DescribeSObjectResult objResult) {
+
+        if (cascades.containsKey(objResult.getSObjectType())) {
+            for (String childLookup : cascades.get(objResult.getSObjectType())) {
+                try {
+                    cascadeDelete(newlist, oldlist, childLookup, triggerAction);
+                } catch (Exception ex) {
+                    ERR_Handler.processError(ex, 'TDTM');
```

I didn't think we'd want to throw an exception for an operation that would have succeeded but left behind orphaned records previously. Let's discuss.